docs/99_developer_guide.md

```diff
@@ -2,35 +2,34 @@
 
 You can use a dev container in VSCode to ease development.
 
-### A quickstart video tutorial on how to set up your environment 
+### A quickstart video tutorial on how to set up your environment
 [![Devcontainer quickstart guide](http://img.youtube.com/vi/h05ufOsRgZU/0.jpg)](http://www.youtube.com/watch?v=h05ufOsRgZU "Devcontainer quickstart guide").
 
 
 ### To deploy:
 #### SC64
-* Download the deployer [here](https://github.com/Polprzewodnikowy/SummerCart64/releases/download/v2.20.0/sc64-deployer-windows-v2.20.0.zip)
+* Download the deployer [here](https://github.com/Polprzewodnikowy/SummerCart64/releases/download/v2.20.0/sc64-deployer-windows-v2.20.0.zip).
 * Extract and place `sc64deployer.exe` in the `tools/sc64` directory.
 
-Make sure that your firmware is compatible (currently v2.20.0+)
+Make sure that your firmware is compatible (currently v2.20.0+). 
```

Comment:
```suggestion
Make sure that your firmware is compatible (currently v2.20.0+).
```

Reply:
Actually needs an extra space to enforce the line break. Either that or rewriting the whole section.

Reply:
On second thought, I'll leave this single-lined.